.github/workflows/ci.yml

```
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - master
+    tags:
+      - 'v[0-9].[0-9]+.[0-9]+'
```

```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+'
```
Might want to add in the `+` for the major version as well so that we don't have to update this again when we reach v10+ (hopefully 😆)

---

Great catch!